connectors/tests/ftest.sh

```diff
@@ -74,3 +69,16 @@ NUM_DOCS=`$PYTHON fixture.py --name $NAME --action get_num_docs`
 $PYTHON $ROOT_DIR/scripts/verify.py --index-name search-$NAME --service-type $NAME --size $NUM_DOCS
 $PYTHON fixture.py --name $NAME --action stop_stack
 $PYTHON fixture.py --name $NAME --action teardown
+
```

Comment:
This should be added before line 61

Reply:
Probably not - Perf8 generates report in the background, so we can save some time here by doing second ingestion and report generation at the same time. Performance of 2nd ingestion does not matter too much, so we can do it that way IMO